src/main/java/picard/sam/FilterSamReads.java

```diff
@@ -205,7 +218,7 @@ public String getHelpDoc() {
     @Argument(
             doc = "Create <OUTPUT>.reads file containing names of reads from INPUT and OUTPUT (for debugging purposes.)",
             optional = true)
-    public boolean WRITE_READS_FILES = true;
+    public boolean WRITE_READS_FILES = false;
```

Comment:
I have no idea why this defaults to true...and I agree with you that it should default to false...however, that is a change to the default behaviour, and thus an api change...I don't think that it's appropriate to put it into this PR...

Please open an issue  and/or a separate PR to address this. (and remove it from this PR.)

Reply:
will do.  I think i changed this because it was being annoying when i was testing and just forgot i made the change